core/src/test/scala/com/salesforce/op/filters/FeatureDistributionTest.scala

```diff
@@ -72,30 +67,47 @@ class FeatureDistributionTest extends FlatSpec with PassengerSparkFixtureTest wi
     distribs(1).nulls shouldBe 1
     distribs(1).distribution.sum shouldBe 0
     distribs(2).distribution.sum shouldBe 2
-    distribs(2).summaryInfo should contain theSameElementsAs Array(0.0, 3.0)
+    distribs(2).summaryInfo should contain theSameElementsAs Array(0.0, 3.0, 7.0, 10.0)
     distribs(3).distribution.sum shouldBe 0
     distribs(4).distribution.sum shouldBe 3
     distribs(4).summaryInfo.length shouldBe bins
   }
 
+  it should "be correctly created for text features" in {
+    val features = Array(description, gender)
+    val values: Array[(Boolean, ProcessedSeq)] = Array(
+      (false, Left(RandomText.strings(1, 10).take(10000).toSeq.map(_.value.get)))
+    )
+    val summary = Array(Summary(1000.0, 50000.0, 70000.0, 10))
+    val bins = 100
+    val featureKeys: Array[FeatureKey] = features.map(f => (f.name, None))
+    val processedSeqs: Array[Option[ProcessedSeq]] = values.map { case (isEmpty, processed) =>
+      if (isEmpty) None else Option(processed)
+    }
+    val distribs = featureKeys.zip(summary).zip(processedSeqs).map { case ((key, summ), seq) =>
+      FeatureDistribution(key, summ, seq, bins, (_, bins) => bins)
+    }
+
+    distribs(0).distribution.length shouldBe 100
+    distribs(0).distribution.sum shouldBe 10000
+
+  }
+
   it should "be correctly created for map features" in {
     val features = Array(stringMap, numericMap, booleanMap).map(TransientFeature.apply)
     val values: Array[Map[String, ProcessedSeq]] = Array(
       Map("A" -> Left(Seq("male", "female"))),
       Map("A" -> Right(Seq(1.0)), "B" -> Right(Seq(1.0))),
       Map("B" -> Right(Seq(0.0))))
-    val summary = Array(
-      Map("A" -> Summary(0.0, 1.0), "B" -> Summary(0.0, 5.0)),
-      Map("A" -> Summary(-1.6, 10.6), "B" -> Summary(0.0, 3.0)),
-      Map("B" -> Summary(0.0, 0.0)))
+    val summaries = Array(
+      Map("A" -> Summary(0.0, 2.0, 100.0, 10), "B" -> Summary(0.0, 5.0, 10.0, 10)),
```

Comment:
Why did the max change for just the `A` key here?

Reply:
This is hardcoded, I changed it since "A" is "male", "female", so it should be 2 instead of 1